app/models/concerns/push_updates.rb

```diff
@@ -33,8 +33,8 @@ def update_client_store(operation)
 
   def browse_updates(operation)
     operation = is_browse? ? operation : "delete"
-    json = Api::V1::BrowsePackageSerializer.new(self).as_json[:browse_package]
-    data = { item: { package: json }, operation: operation }
+    json = Api::V1::BrowsePackageSerializer.new(self).as_json
+    data = { item: { package: json[:browse_package], images: json[:images]  }, operation: operation }
```

Comment:
@steveyken , BrowseItemSerializer is just used to pass the parent-item details(package-type, donor-condition). We only send package-details to browse app using  BrowsePackageSerializer and item-details are send via BrowsePackageSerializer.
Here while sending updates we only need updates of package and images hence using BrowsePackageSerializer